src/pages/form-validation/password/index.tsx

```diff
@@ -0,0 +1,435 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import {
+    AppBar,
+    Toolbar,
+    IconButton,
+    Typography,
+    TextField,
+    InputAdornment,
+    List,
+    ListItem,
+    InputProps,
+    Hidden,
+    Divider,
+    Button,
+    Card,
+} from '@material-ui/core';
+import MenuIcon from '@material-ui/icons/Menu';
+import { Done, Visibility, VisibilityOff } from '@material-ui/icons';
+import { makeStyles, useTheme, Theme } from '@material-ui/core/styles';
+import { useDispatch } from 'react-redux';
+import { TOGGLE_DRAWER } from '../../../redux/actions';
+import * as Colors from '@pxblue/colors';
+import { Spacer } from '@pxblue/react-components';
+import clsx from 'clsx';
+
+const useStyles = makeStyles((theme: Theme) => ({
+    containerWrapper: {
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+        flex: '1 1 0',
+    },
+    container: {
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        backgroundColor: 'white',
+        paddingTop: theme.spacing(4),
+        paddingLeft: theme.spacing(3),
+        paddingRight: theme.spacing(3),
+        width: 450,
+        [theme.breakpoints.down('xs')]: {
+            width: '100%',
+            minHeight: 'calc(100vh - 56px)',
+            paddingTop: theme.spacing(2),
+            paddingLeft: theme.spacing(2),
+            paddingRight: theme.spacing(2),
+        },
+    },
+    sectionHeader: {
+        marginBottom: 16,
+        [theme.breakpoints.down('xs')]: {
+            display: 'none',
+        },
+    },
+    newPasswordInputField: {
+        marginTop: theme.spacing(5.5),
+        [theme.breakpoints.down('xs')]: {
+            marginTop: theme.spacing(3),
+        },
+    },
+    block: {
+        display: 'flex',
+        flexDirection: 'column',
+        flex: 1,
+        maxWidth: 600,
+        paddingBottom: 0,
+    },
+    passwordCriteria: {
+        paddingTop: 0,
+        paddingBottom: 0,
+    },
+    appbarRoot: {
+        padding: 0,
+    },
+    toolbarGutters: {
+        padding: '0 16px',
+    },
+    divider: {
```

Comment:
The divider should have top margin 40, bottom margin 32 according to the specs on doc it (the design file is incorrect in this regard)